mlir/docs/Dialects/Vector.md

```diff
@@ -286,10 +299,11 @@ Consider a vector of rank n with static sizes `{s_0, ... s_{n-1}}` (i.e. an MLIR
 `vector<s_0x...s_{n-1}xf32>`). Lowering such an `n-D` MLIR vector type to an
 LLVM descriptor can be done by either:
 
-1.  Flattening to a `1-D` vector: `!llvm<"(s_0*...*s_{n-1})xfloat">` in the MLIR
+1.  Nested aggregate type of `1-D` vector:
```

Comment:
is the order here is important?

Reply:
If you scroll further down, you will see that Option 1 and Option 2 are discussed in the reverse order. I just wanted to improve the flow (also got confused myself when reading this). 

I can move this change to a separate patch, but am concerned that that's just extra noise.